.travis.yml

```diff
@@ -35,7 +33,7 @@ before_install:
   - sudo easy_install pip
   - sudo pip install virtualenv
   # Python 2.7 support for SNI (https://stackoverflow.com/questions/18578439/using-requests-with-tls-doesnt-give-sni-support/18579484#18579484)
-  - sudo pip install urllib3 pyopenssl ndg-httpsclient pyasn1
+  - sudo pip install pyopenssl ndg-httpsclient pyasn1
```

Comment:
I noticed the build is timing out. I think we may be able to get around that by splitting build and test into two different stages: https://docs.travis-ci.com/user/build-stages/ (each stage is allowed to go for 2 hours independently).

Reply:
Except there is a bit of a hiccup around persistence. Stages don't seem that useful since we can't share data easily across them: https://docs.travis-ci.com/user/build-stages/#data-persistence-between-stages-and-jobs